exercises/twelve-days/twelve-days.spec.js

```diff
@@ -0,0 +1,106 @@
+    expect(twelveDaysObject.verse(7)).toEqual(expectedVerseSeven);
+  });
+
+  xtest('test verse eight', () => {
+    const expectedVerseEight =
+      'On the eighth day of Christmas my true love gave to me, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n';
+    expect(twelveDaysObject.verse(8)).toEqual(expectedVerseEight);
+  });
+
+  xtest('test verse nine', () => {
+    const expectedVerseNine =
+      'On the ninth day of Christmas my true love gave to me, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n';
+    expect(twelveDaysObject.verse(9)).toEqual(expectedVerseNine);
+  });
+
+  xtest('test verse ten', () => {
+    const expectedVerseTen =
+      'On the tenth day of Christmas my true love gave to me, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n';
+    expect(twelveDaysObject.verse(10)).toEqual(expectedVerseTen);
+  });
+
+  xtest('test verse eleven', () => {
+    const expectedVerseEleven =
+      'On the eleventh day of Christmas my true love gave to me, eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n';
+    expect(twelveDaysObject.verse(11)).toEqual(expectedVerseEleven);
+  });
+
+  xtest('test verse twelve', () => {
+    const expectedVerseTwelve =
+      'On the twelfth day of Christmas my true love gave to me, twelve Drummers Drumming, eleven Pipers Piping, ten Lords-a-Leaping, nine Ladies Dancing, eight Maids-a-Milking, seven Swans-a-Swimming, six Geese-a-Laying, five Gold Rings, four Calling Birds, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n';
+    expect(twelveDaysObject.verse(12)).toEqual(expectedVerseTwelve);
+  });
+
+  xtest('test multiple verse', () => {
+    const expectedVerseOneToThree =
+      'On the first day of Christmas my true love gave to me, a Partridge in a Pear Tree.\n\n' +
+      'On the second day of Christmas my true love gave to me, two Turtle Doves, and a Partridge in a Pear Tree.\n\n' +
+      'On the third day of Christmas my true love gave to me, three French Hens, two Turtle Doves, and a Partridge in a Pear Tree.\n';
+    expect(twelveDaysObject.verse(1, 3)).toEqual(expectedVerseOneToThree);
+  });
+
+  xtest('test sing whole song', () => {
+    const expectedSong =
+      'On the first day of Christmas my true love gave to me, a Partridge in a Pear Tree.\n\n' +
```

Comment:
@matthewmorgan actually I thought that this was more readable; as this would give others some hint about the kind of approach they may take. Also, I tried multiline back ticks but it, due to indentation, I was getting unexpected respected results. So I went back to the old ways.....
Hope it works....